test/jpeg_sky130hd.metrics

```diff
@@ -2,68 +2,68 @@
+	"design__instance__displacement__mean": 0.051,
+	"design__instance__displacement__max": 11.773,
+	"route__wirelength__estimated": 1.50883e+06,
+	"design__instance__count__setup_buffer": 126,
+	"design__instance__count__hold_buffer": 84,
+	"RSZ::worst_slack_min": "0.00221644930904713",
+	"RSZ::worst_slack_max": "-0.05123634994630406",
+	"RSZ::tns_max": "-0.8139027460146046",
+	"RSZ::hold_buffer_count": "84",
+	"design__instance__displacement__total": 1865.27,
+	"design__instance__displacement__mean": 0.025,
+	"design__instance__displacement__max": 13.904,
+	"route__wirelength__estimated": 1.56465e+06,
+	"DPL::utilization": "23.6",
+	"DPL::design_area": "515658",
+	"route__net": 57288,
 	"route__net__special": 2,
-	"antenna__violating__nets": 4,
-	"antenna__violating__pins": 5,
-	"GRT::ANT::errors": "4",
+	"antenna__violating__nets": 2,
+	"antenna__violating__pins": 3,
+	"GRT::ANT::errors": "3",
 	"design__violations": 0,
-	"route__net": 57197,
+	"route__net": 57288,
 	"route__net__special": 2,
-	"route__drc_errors__iter:1": 14683,
-	"route__wirelength__iter:1": 1755780,
-	"route__drc_errors__iter:2": 1397,
-	"route__wirelength__iter:2": 1746065,
-	"route__drc_errors__iter:3": 655,
-	"route__wirelength__iter:3": 1743952,
-	"route__drc_errors__iter:4": 8,
-	"route__wirelength__iter:4": 1743892,
+	"route__drc_errors__iter:1": 14426,
+	"route__wirelength__iter:1": 1755106,
+	"route__drc_errors__iter:2": 1320,
+	"route__wirelength__iter:2": 1745084,
+	"route__drc_errors__iter:3": 736,
+	"route__wirelength__iter:3": 1742887,
+	"route__drc_errors__iter:4": 64,
+	"route__wirelength__iter:4": 1742817,
 	"route__drc_errors__iter:5": 0,
-	"route__wirelength__iter:5": 1743893,
+	"route__wirelength__iter:5": 1742830,
 	"route__drc_errors": 0,
-	"route__wirelength": 1743893,
-	"route__vias": 307603,
-	"route__vias__singlecut": 307603,
+	"route__wirelength": 1742830,
+	"route__vias": 308890,
+	"route__vias__singlecut": 308890,
 	"route__vias__multicut": 0,
 	"DRT::drv": "0",
-	"antenna__violating__nets": 46,
-	"antenna__violating__pins": 50,
-	"DRT::ANT::errors": "46",
+	"antenna__violating__nets": 37,
+	"antenna__violating__pins": 40,
+	"DRT::ANT::errors": "37",
 	"timing__drv__floating__nets": 0,
 	"timing__drv__floating__pins": 0,
-	"DRT::worst_slack_min": "-0.07115219725910353",
-	"DRT::worst_slack_max": "-0.7011751377981101",
-	"DRT::tns_max": "-106.38896287291976",
-	"DRT::clock_skew": "0.5462244144933402",
-	"DRT::max_slew_slack": "-7.390156886595937",
+	"DRT::worst_slack_min": "-0.14849411010015434",
+	"DRT::worst_slack_max": "-0.3264864027096548",
+	"DRT::tns_max": "-12.091829213499114",
+	"DRT::clock_skew": "0.5987970289098753",
+	"DRT::max_slew_slack": "-1.8637025085292298",
 	"DRT::max_fanout_slack": "100.0",
-	"DRT::max_capacitance_slack": "-9.419548319063965",
+	"DRT::max_capacitance_slack": "-4.683004334395966",
 	"DRT::clock_period": "8.000000",
-	"flow__warnings__count": 22,
+	"flow__warnings__count": 1634,
```

Comment:
What is the cause of this increase?

Reply:
I see 1000 warnings on "Missing route to pin _xxx_/A", I believe it is during GRT:

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/37420320/c62ab18b-9ed9-498c-8a8d-37b8a73940b3)

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/37420320/171b60e5-0769-4cc2-bd53-8bda2b8f0dd0)

I see no "Missing route..." warning on the master log.

Reply:
That seems odd.  Please look into it.  @luis201420 can help if needed.

Reply:
These relate to the extraction in GRT and are caused by a "poor" definition of a pin and how a guide connects to it. (it would be amazing if these warnings could be removed since they are very confusing when users first see them)

Reply:
That's the plan

Reply:
Hi @gadfort and @maliberty , thanks for your suggestion. Yesterday me and @arthurjolo were trying to fix the issue behind this warning. Indeed there seem to be a mismatch between the pin position accessed on MakeWireParasitics.cpp and the pin position seen by fastroute. 

We found something strange in the code: in `MakeWireParasitics::makeParasiticsToPin(` it tries to get the pin position with an access point, although it seems fastroute does not consider access points either way.

So there might be an actual error behind this warnings. Either way, all that happens is that we will be missing a parasitics value for the pins with mismatch positions.

Reply:
I recently created a PR to remove this part of the code: #5155 

Should I also remove the warning massage completely?

Reply:
No, it did point to a real problem.